lib/mixins/dropdown.js

```diff
@@ -81,6 +80,9 @@ export default {
                         el.addEventListener('mouseover', this.noop);
                     });
                 }
+
+                document.addEventListener('keydown', this.keydownListener);
```

Comment:
Hmm sorry i didn't know that 😞  Was just tried to follow current BS4-Beta approach as they don't make that focus still stating ARIA compatible :)) So you think it would be better revert that?